modelservice/src/main/sql/build-db.sql

```diff
@@ -0,0 +1,74 @@
+DROP DATABASE IF EXISTS model_service;
+
+CREATE DATABASE model_service;
+
+USE model_service;
+
+
+CREATE TABLE metric
+(
+  metric_id   INT UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  metric_uuid CHAR(32) UNIQUE          NOT NULL
+);
+
+CREATE TABLE models
```

Comment:
For consistency, we should either always use singular or always use plural. (So not "metric" for one table" and "models" for another.) My request would be that we just use singular for the table names.